### PR TITLE
Fix -Wsign-conversion Clang warning.

### PR DESCRIPTION
**Pull request by robertsipka**
JerryScript-DCO-1.0-Signed-off-by: Robert Sipka rsipka.uszeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
LGTM

File: jerry-libc/jerry-libc.c [hunk 1]
```diff
@@ -267,7 +267,7 @@ rand (void)
   libc_random_gen_state[3] ^= libc_random_gen_state[3] >> 19;
   libc_random_gen_state[3] ^= intermediate;
 
-  return libc_random_gen_state[3] % (RAND_MAX + 1u);
+  return (int) (libc_random_gen_state[3] % (RAND_MAX + 1u));
```
Comment (zherczeg) [inline jerry-libc/jerry-libc.c]: What about changing 1u to 1?

Comment (robertsipka) [inline jerry-libc/jerry-libc.c]: Ohh, I missed it. Thanks.

File: jerry-libc/jerry-libc.c [hunk 2]
```diff
@@ -267,7 +267,7 @@ rand (void)
   libc_random_gen_state[3] ^= libc_random_gen_state[3] >> 19;
   libc_random_gen_state[3] ^= intermediate;
 
-  return libc_random_gen_state[3] % (RAND_MAX + 1u);
+  return (int) (libc_random_gen_state[3] % (RAND_MAX + 1));
```
Comment (zherczeg) [inline jerry-libc/jerry-libc.c]: Do we still need the cast? Can libc_random_gen_state[3] be int?

Comment (zherczeg) [inline jerry-libc/jerry-libc.c]: It seems no.

**Comment by zherczeg** [comment]
LGTM

**Comment by robertsipka** [comment]
Thanks, I've rebased it with master.

**Pull request merged**